xarray/core/pdcompat.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+import pandas as pd
+import pandas.core.algorithms as algos
```

Comment:
can we do a local import here instead the function?

I'm a little nervous that some future version of pandas may drop this (private) module, which would then break imports even though this isn't actually used.

Reply:
Good catch! Fixed.